resources/assets/js/store/modules/site.js

```diff
@@ -12,7 +12,9 @@ const state = {
 	pageModal: {
 		visible: false,
 		parentId: null
-	}
+	},
+	sitePath: 'site',
+	siteDomain: 'https://webtools-test.kent.ac.uk/'
```

Comment:
What are these for? sitePath and siteDomain?

We should not be hard-coding Kent-specific stuff like this.

We should also start documenting (even if a couple of inline comments) what any properties we are adding are for.

Reply:
@sy238-unikent can we add these to the api so we can pull them out for each page/site?

Reply:
- added comments to the state variables
- behaviour is a bit broken till we can get the frontend domain and site page. This isn't just in the title bar, but also means we can't link people to the correct url to view the page.